DuckDuckGo/OmniBar.swift

```diff
@@ -300,10 +301,15 @@ class OmniBar: UIView {
             showCustomIcon(icon: .duckPlayer)
             return
         }
-        
-        privacyInfoContainer.privacyIcon.isHidden = privacyInfo.isSpecialErrorPageVisible
+
+        if privacyInfo.isSpecialErrorPageVisible {
+            showCustomIcon(icon: .specialError)
+            return
+        }
```

Comment:
how does it look like for ssl error page now? I remember decision was not to show any icon, are we good there?